tests/bluetooth/tester/src/l2cap.c

```diff
@@ -150,7 +150,7 @@ static void connect(uint8_t *data, uint16_t len)
 	uint8_t buf[sizeof(*rp) + 1];
 	int err;
 
-	if (cmd->num > 1 || mtu > DATA_MTU) {
+	if (cmd->num > 1 || mtu > DATA_BUF_SIZE) {
```

Comment:
Since the name has been updated it seems that there was a problem here as well.
MTU > (DATA_BUF_SIZE - BT_L2CAP_CHAN_SEND_RESERVE) like in other places.
Maybe even add a define for it.

`#define DATA_MTU  (DATA_BUF_SIZE - BT_L2CAP_CHAN_SEND_RESERVE) `